pkg/features/kube_features.go

```diff
@@ -889,7 +890,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	TTLAfterFinished:                               {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.25
 	IndexedJob:                                     {Default: true, PreRelease: featuregate.Beta},
 	JobTrackingWithFinalizers:                      {Default: true, PreRelease: featuregate.Beta},
-	JobReadyPods:                                   {Default: false, PreRelease: featuregate.Alpha},
+	JobReadyPods:                                   {Default: true, PreRelease: featuregate.Beta},
```

Comment:
I thought we were no longer defaulting beta features on?

Reply:
beta endpoints, to start, since that is what causes 99% of user pain.

policy for beta fields hasn't changed, since they have proven far less disruptive.

Reply:
OK